src/common/utils/mediaHelpers.js

```diff
@@ -22,7 +22,7 @@ export const getMediaPath = ({ mediaObject, type, size = ORIGINAL }) => {
 
     case VIDEO:
     case IMAGE:
-      return mediaObject?.[size]?.path
+      return mediaObject?.[size]?.path || mediaObject?.original?.path
```

Comment:
Will this cause any problems for videos? If the caller is expecting a thumbnail but gets a video path, they won't be able to display it in an img tag.

Reply:
Good call! Thanks :)